actions/java/com/squareup/tools/sqldelight/cli/Args.kt

```diff
@@ -40,6 +40,13 @@ object Args : Validating {
   )
   var moduleName: String? = null
 
+  @Parameter(
+          names = ["--database_name"],
+          required = false,
+          description = "Database Name for Kotlin compilation (not required for legacy). Default as `Database`."
+  )
+  var databaseName: String? = null
```

Comment:
I think you can make this String and just set "Database" here as the default. Then you can avoid the `:?` later on.

Reply:
@cgruber updated

Reply:
reping @cgruber for review

Reply:
Sorry - got backed up.